pydanticrud/backends/dynamodb.py

```diff
@@ -217,16 +212,16 @@ def __init__(self, cls, result, serialized_items):
 
 class Backend:
     def __init__(self, cls):
-        cfg = cls.model_config
+        cfg = cls.DBConfig
         self.cls = cls
         self.schema = cls.schema()
-        self.hash_key = cfg.get("hash_key")
-        self.range_key = cfg.get("range_key")
-        self.serializer = DynamoSerializer(self.schema, ttl_field=cfg.get("ttl"))
+        self.hash_key = cfg.hash_key
+        self.range_key = getattr(cfg, "range_key", None)
```

Comment:
```suggestion
        self.range_key = getattr(cfg, "range_key")
```
The `None` is redundant here right?

Reply:
Its required otherwise we get AttributeError.